packages/components/src/custom-gradient-picker/types.ts

```diff
@@ -26,6 +26,10 @@ export type CustomGradientPickerProps = {
 	 * the `currentGradient` as an argument.
 	 */
 	onChange: ( currentGradient: string ) => void;
+	/**
+	 * Whether to disable alpha transparency options in the picker.
+	 */
+	disableAlpha?: boolean;
```

Comment:
`disableAlpha` was deprecated for the underlying `ColorPicker`, but `CustomGradientPicker` correctly uses the proper `enableAlpha` prop, so we should be good. 

I wonder if we intended to follow the same pattern for this `disableAlpha` prop, @ciampo @mirka?

Reply:
> The only thing I'm worried about is the prop name.  `ColorPicker`'s `disableAlpha` prop is deprecated in favor of the opposite `enableAlpha` prop. Ideally it would be nice to follow the same pattern for all components that use `ColorPicker` under the hood. Whether we want to do it here or in another PR

Yes, I think we should flip it to be `enableAlpha`, if only for consistency between relevant components. And we should do it in this PR.

Reply:
If we flip the prop to enableAlpha, what should the default value be? ColorPicker defaults it to false, but GradientPicker was always set to have alpha enabled.

Reply:
We don't want to change the default behavior, so the default should be `true` 👍

Reply:
In that case, I'll make the flip within GradientPicker component, and leave disableAlpha prop in custom-gradient-picker and custom-gradient-bar as is.

Reply:
OK, the changes are made.

I changed the prop name to enableAlpha in both GradientPicker and custom-gradient-picker components and the flip is done within CustomGradientPicker. I left the disableAlpha prop as it was in CustomGradientBar.

Note that I rolled back previous commits and did everything in one go instead while also updating and rebasing trunk. However, I am not sure that I did everything correctly.